sql/parser/datum.go

```diff
@@ -31,26 +31,26 @@ import (
 
 var (
 	// DummyBool is a placeholder DBool value.
-	DummyBool = DBool(false)
+	DummyBool = Datum(DBool(false))
```

Comment:
I think I'd prefer the `DummyBool Datum = DBool(false)` syntax, but LGTM either way.